### PR TITLE
Add msgid's AlarmEventList, HDDInfoList, Support

### PR DESCRIPTION
**Pull request by kickbits**
Fix xml_buffer xml check
Update body labels

### CODE REVIEW

**Comment by kickbits** [comment]
Apparently my twistedddx user can not have 2 forks from the same network, eg a fork of 
https://github.community/t/alternatives-to-forking-into-the-same-account/

Eg I can't fork both thirtythreeforty and QuantumEntangledAndy neolink's.
If I tried to do a pull request from my twistedddx fork all my commits are based off thirtythreeforty base.
Something that should be a few lines of changes becomes a complete resubmit of all code.
Basically a merge would bring all of your code back to thirtythreeforty base and then a full change back to what I desired the outcome to be. Almost all lines in the file would be considered altered.
What a pain!

**Comment by QuantumEntangledAndy** [comment]
Seems like it... Is this now all covered in  #3?

**Comment by QuantumEntangledAndy** [comment]
Also you wouldn't need two forks. Just seperate branches.

You would do something like this.

Add an upstream origin

```bash
git remote add-url qeandy https://github.com/QuantumEntangledAndy/neolink
git checkout -t qeandy mqtt_dissector
# make changes
# commit and publish to your fork and PR to me
```

Since this is now in the other PR I will close this one :)
